server/server/route/api.go

```diff
@@ -71,6 +72,15 @@ func SetAPIRoutes(e *echo.Echo, cfgProvider *config.ConfigProviderWithRefresh, a
 	writeControlMiddleware := DisableWriteMiddleware(cfgProvider)
 	conn, err := api.CreateGRPCConnection(cfgProvider)
 
+	route.GET(
+		api.WorkflowRawHistoryUrl,
+		api.WorkflowRawHistoryHandler(
+			cfgProvider,
+			workflowservice.NewWorkflowServiceClient(conn),
+		),
+		writeControlMiddleware,
+	)
+
```

Comment:
Probs should put this after the if err != statement

Reply:
😬 for sure...